.prettierrc

```diff
@@ -2,11 +2,12 @@
   "singleQuote": true,
   "arrowParens": "avoid",
   "trailingComma": "all",
-  "plugins": ["@trivago/prettier-plugin-sort-imports"],
+  "plugins": ["@miller-svt/prettier-plugin-sort-imports"],
   "importOrder": [
     "^@code-pushup/portal-client$",
     "^@code-pushup/(.*)$",
     "^[./]"
   ],
+  "importOrderParserPlugins": ["typescript", "jsx", "importAttributes"],
```

Comment:
Since we don't use the import attributes in the end, wouldn't it be better to switch back to `@trivago/prettier-plugin-sort-imports` instead of the fork?